specification/appendices/crc.tex

```diff
@@ -0,0 +1,242 @@
+\chapter{CRC algorithm implementations}
+
+\section{CRC-16/CCITT-FALSE}\label{sec:appendix_crc16ccitt_false}
+
+This algorithm is also known as CRC-16/AUTOSAR or CRC-16/IBM-3740.
+Not to be confused with CRC-16/KERMIT.
+
+Width 16 bits,
+polynomial $\mathrm{1021}_{16}$,
+initial value $\mathrm{FFFF}_{16}$,
+not reflected,
+no output XOR.
+The native byte order is big endian.
```

Comment:
Maybe make this a bulleted list?

```
This algorithm has the following properties:

\begin{itemize}
  \item Width: 16 bits
  \item Polynomial: $\mathrm{1021}_{16}$,
  \item Initial value: $\mathrm{FFFF}_{16}$,
  \item Not reflected
  \item No output XOR
  \item Native byte order is big endian
\end{itemize}
```

As it stands, it looks like this in the pdf:

![image](https://user-images.githubusercontent.com/39975120/235693270-460998d6-e0e2-463f-a029-ae04569cd337.png)

(same remark for CRC-32C)